controls/datagrid/filtering/programmatic-filtering.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Programmatic Filtering
+page_title: .NET MAUI DataGrid Documentation - Programmatic Filtering
+description: Check our Programmatic Filtering documentation article for Telerik DataGrid for .NET MAUI control.
```

Comment:
Rename description, explain in what cases programmatic filtering can be used. 
for external filtering. For example you disable the built in filtering UI and want to programmatically filter the data in the grid.